docs/languages/_posts/2018-12-05-c.md

```diff
@@ -0,0 +1,22 @@
+---
+title: The C Programming Language
+layout: default
+last-modified: 2018-12-05
+featured-image:
+tags: [c]
+authors:
+  - the_renegade_coder
+---
+As usual, I decided to dive into the history of C with [Wikipedia][1].
+
+According to Wikipedia, C is a general-purpose programming language that was developed as early as 1969 at Bell Labs by Dennis Richie. Today, the C language has influenced the design of many languages such as C++, C#, Go, Java, and many, many more.
+
+Features of C include static typing, lexical variable scope, and recursion. In addition, function parameters are passed by value, not reference, but pass-by-reference can be accomplished by explicitly using pointers. Also, while C has static typing, variables are weakly typed and can be converted implicitly.
+
+---
```

Comment:
Check the template, but I believe there's a missing section here (`## Articles`).